### DISCUSSION TITLE
Me pueden compartir el link de la sesión de hoz por favor

### DISCUSSION THREAD

**Discussion by JaimeEsqueda** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#6] [General]
No lo veo en teams ni por aqui, una disculpa de ante mano, agradezco mucho su tiempo y atencion

**Comment by OscarPedroza** [comment]
https://teams.microsoft.com/l/meetup-join/19%3ameeting_MTA2YmZkMjMtYmFiMS00ZmRkLTk1YmQtYWVmMWMzYjA0ZGMz%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Comment by MiguelSoberanoCarranza** [comment]
No se por que no aparecen y si sea ese lo encontré en los comentarios de la primera sesión. 

https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZTgyN2JhMGYtN2QwNi00NWFmLTk0NzItY2ExY2NkOWMxNTMx%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Comment by Edgarsantome** [comment]
Hola 

Donde puedo ver la sesión del día de hoy, ya que tampoco la encuetro

**Comment by JonathanGarcia15** [comment]
Pues creo que hoy no hubo sesión porque no me apareció en absoluto nada. 🙃

**Comment by doguedogue** [comment]
1. Entras al teams con tu cuenta de @innovaccion
2. En la parte de equipos
![image](https://user-images.githubusercontent.com/23409026/154528758-c069f996-f179-4088-9385-77c6e9dd873e.png)

3. Semana de despegue (o semana actual)
![image](https://user-images.githubusercontent.com/23409026/154528980-f727a9f7-5318-4409-af11-5218fedcc051.png)

4. Ver las publicaciones y las fechas
5. Clic en ver mas (para expandir la publicación) y buscar el link
![image](https://user-images.githubusercontent.com/23409026/154529421-53068279-42cc-4692-af98-93b6bf18a1b3.png)

**Reply by JoseMendieta85** [reply]
Hola, yo tampoco puedo ver el link de la sesion de ayer miercoles 16 y tampoco me aparece ni la opcion que presentas y la liga que comparten arriba me redirige a la sesion de lunes. A alguien mas le pasa?

<img width="862" alt="image" src="https://user-images.githubusercontent.com/23405057/154539776-abfe8c4f-ce51-45c3-91a2-f938358687a2.png">

**Comment by JoseMendieta85** [comment]
Ya encontre una liga:

https://web.microsoftstream.com/video/d02d524e-8bbb-466e-acc1-13407fd12747

Saludos.

**Comment by AliciaGaona** [comment]
Hola compañeros, tampoco encontraba la grabación de ayer, y la pude encontrar aquí, espero les sirva


https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645037927085?groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0
